platform/linux-generic/arch/aarch64/odp_cpu_idling.h

```diff
@@ -19,6 +19,8 @@
 
 #else /* CONFIG_WFE */
 
+#include <odp/api/cpu.h>
+
```

Comment:
It's better to remove doze() altogether, since it's always implemented as odp_cpu_pause(). Doze is called only in two files: odp_queue_scalable.c and odp_schedule_scalable.c